Source/Core/DolphinQt/GameList/GameListModel.cpp

```diff
@@ -140,6 +141,26 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
       return QString::fromStdString(game.GetFileName());
     break;
+  case COL_FILE_PATH:
+    if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
+    {
+      QString file_path = QDir::cleanPath(QString::fromStdString(game.GetFilePath()));
```

Comment:
I'm a little concerned about the performance impact of the logic here when used with very large game lists. But, I guess if the column is hidden by default, it shouldn't get executed.

Reply:
How do you suggest I improve the performance impact?

Reply:
Would probably require caching the string in the game list struct. But again, since it's not enabled by default, I don't think it matters all that much.